### DISCUSSION TITLE
Can't turn off unwanted wifi connection

### DISCUSSION THREAD

**Discussion by ceejayemm** [rern/rAudio#356] [Bug Reports]
PI3B
wired connection only

After updating to 20210530 I now find I have two networks active, the wired connection which I normally use and a wireless connection on another IP number which I don't want.

From the System page if I try to turn off the wireless connection via the WiFi option I get a small red and white 'no entry' icon and I cannot make the change.

Chris

**Comment by Roland-B** [comment]
@ceejayemm Have you checked "Access Point" has not been enabled somehow? I had the same issue and discovered it had turned itself on.

RolandB

**Reply by ceejayemm** [reply]
Roland

Thanks for you input. No Access Point is not enabled and never has been as I have no need to use it.

Chris
